osmosis-1/osmosis-1.assetlist.json

```diff
@@ -948,6 +948,23 @@
       "logo_URIs": {
         "png": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/dig.png"
       }
+    },
+    {
+      "description": "Rowan Token (ROWAN) is the Sifchain Network's native utility token, used as the primary means to govern, provide liquidity, secure the blockchain, incentivize participants, and provide a default mechanism to store and exchange value.",
+      "denom_units": [
+        {
+          "denom": "rowan",
```

Comment:
This needs to be the `ibc/HASH` denom name

Reply:
Updated  ✅